cypress/test_objects/dresses-list-objects.ts

```diff
@@ -0,0 +1,11 @@
+const expectedDressesList = [
+  "Printed Dress",
+  "Printed Dress",
+  "Printed Summer Dress",
+  "Printed Summer Dress",
+  "Printed Chiffon Dress",
+];
+
+const numberOfDresses = 5;
+
+export {expectedDressesList, numberOfDresses};
```

Comment:
as this will only be used in one test, it does not makes sense to create a separated file with this configuration. Also, this makes hard to read what is the value we want to assert or verify in the tests.